config.dev.template.toml

```diff
@@ -1,11 +1,20 @@
-token = ""
-ownerIds = [1234]
-prefixes = []
-testGuildIds = [1234]
+token = "XXXXXXXXXXXXXXXXXXXXXXXX.XXXXXX.XXXXXX_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+ownerIds = [
+    00000000000000000,
+    00000000000000000,
+]
+prefixes = [
+    "!",
+    "?",
+]
+testGuildIds = [
+    00000000000000000,
+    00000000000000000,
+]
```

Comment:
what the fuck?

Reply:
For token you can just do `<BOT_TOKEN>` or something and for the ids `1234` is fine

Reply:
Alright